<launch>

  <param name="use_sim_time" value="false"/>

  <include file="$(find sick_scan)/launch/sick_lms_1xx.launch"/>

  <node pkg="tf" type="static_transform_publisher" name="laser" args="0.5 0 0 0 0 0 map laser 30" />

  <node pkg="sick_filter" type="sick_vehicle_roi" name="sick_vehicle_roi_node"/>

  <node pkg="sick_filter" type="sick_cluster" name="sick_cluster_node">
    <param name="x_axis_min" value="-0.0"/>
    <param name="x_axis_max" value="5.0"/>
    <param name="y_axis_min" value="-1.5"/>
    <param name="y_axis_max" value="1.5"/>

    <param name="cluster_meter" value="0.3"/>
    <param name="cluster_size_min" value="3"/>
    <param name="cluster_size_max" value="5000"/>    
  </node>

  <node pkg="sick_filter" type="sick_dynamic" name="sick_dynamic_node" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find sick_filter)/resources/sick_cluster.rviz"/>

</launch>
